Sources/Sonar/Models/ModelsData.swift

```diff
@@ -65,21 +65,61 @@ extension Area {
+        Area(13, "Disk Utility"), Area(14, "Dock"), Area(15, "Documentation"), Area(16, "FaceTime"),
+        Area(17, "Final Cut Pro"), Area(18, "Finder"), Area(19, "Graphics & Imaging"),
+        Area(20, "iBooks / Author"), Area(21, "iCloud / CloudKit"), Area(22, "Installation"),
+        Area(23, "iTunes"), Area(24, "iTunes Connect"), Area(25, "Java"),
+        Area(26, "Keyboards, mice and trackpads"), Area(27, "Keynote, Numbers & Pages"),
+        Area(28, "Launchpad"), Area(29, "Localization"), Area(30, "Logic Pro X"), Area(31, "Mail"),
+        Area(32, "Maps"), Area(33, "Menu Bar"), Area(34, "Messages"), Area(35, "Mission Control"),
+        Area(36, "Networking"), Area(37, "News Publisher"), Area(38, "Notes"), Area(39, "Numbers"),
+        Area(40, "Performance (slow, hung"), Area(41, "Photos"), Area(42, "Preferences"), Area(43, "Preview"),
+        Area(44, "Printing & Faxing"), Area(45, "Reminders"), Area(46, "Safari"), Area(47, "App Sandbox"),
+        Area(48, "Screen Saver & desktop"), Area(49, "Server (macOS Server"), Area(50, "Setup Assistant"),
+        Area(51, "Software Update"), Area(52, "Spaces"), Area(53, "Spotlight"), Area(54, "Swift"),
+        Area(55, "Terminal"), Area(56, "Time Machine"), Area(57, "Wi-Fi"), Area(58, "Xcode"),
+        Area(59, "Something not on this list"),
+    ]
+
+    fileprivate static let AlltvOSAreas: [Area] = [
+        Area(1, "Accessibility"), Area(2, "Accounts"), Area(3, "AirPlay"), Area(4, "App Store"),
+        Area(5, "App Switcher"), Area(6, "AV Playback"), Area(7, "Bluetooth"),
+        Area(8, "Device Management / Profiles"), Area(9, "Display"), Area(10, "Home Screen"),
+        Area(11, "Movies / TV Shows"), Area(12, "Music"), Area(13, "Networking"), Area(14, "Photos"),
+        Area(15, "Remote.app Support"), Area(16, "Setup (includes Tap Setup"), Area(17, "Screensaver"),
+        Area(18, "Search"), Area(19, "Settings"), Area(20, "Siri"), Area(21, "Software Update"),
+        Area(22, "SSO"), Area(23, "System, General"), Area(24, "TV App"), Area(25, "TVML Apps"),
+        Area(26, "tvOS SDK"),
+    ]
+
+    fileprivate static let AllwatchOSAreas: [Area] = [
+        Area(1, "Accessibility"), Area(2, "Activity"), Area(3, "Battery Life"), Area(4, "Bluetooth"),
+        Area(5, "Calendar"), Area(6, "Contacts"), Area(7, "Control Center"), Area(8, "Dock"),
+        Area(9, "HealthKit"), Area(10, "HomeKit"), Area(11, "Maps"), Area(12, "Messages"), Area(13, "Music"),
+        Area(14, "Notification Center"), Area(15, "Notifications"), Area(16, "Setup and Pairing"),
+        Area(17, "Simulator"), Area(18, "Siri"), Area(19, "Syncing"), Area(20, "Wallet"),
+        Area(21, "Watch Faces"), Area(22, "WatchKit"), Area(23, "Workout"), Area(24, "Xcode"),
+    ]
 }
 
 extension Product {
-
-    private init(_ appleIdentifier: Int, _ name: String, _ category: String) {
+    private init(_ appleIdentifier: Int, _ name: String, _ category: String, _ areas: [Area] = []) {
```

Comment:
confused about this part why would a product have the entire array of areas? I mean the `Product` is also part of a `Radar`, so it doesn't make much sense to send all areas, does it? I guess I'm missing something here?

Say that I want to create a `Radar` value:

```swift
Radar(title: "Hello", product: .iOS)
```

What would the array of areas mean here for the product `.iOS`? I'd expect this to be:

```swift
Radar(title: "Hello", product: .iOS, area: .siri)
```

I think instead of having areas into the product model what you want is a `Area.areasByProductID` kind of hash map.